app/components/navbar.hbs

```diff
@@ -0,0 +1,33 @@
+<nav class="nav">
+  <button type="button" class="hamburger--toggle" {{on "click" this.toggleNav}}>
+    <span class="hamburger__bar"></span>
+    <span class="hamburger__bar"></span>
+    <span class="hamburger__bar"></span>

```

Comment:
I believer this is for the hamburger menu. We are already having font-awesome library in our project, can you please check if they provide an icon for the same?

Reply:
ok great will check